packages/grafana-ui/src/components/Link/TextLink.story.tsx

```diff
@@ -0,0 +1,71 @@
+import { Meta, StoryFn } from '@storybook/react';
+import React from 'react';
+
+import { StoryExample } from '../../utils/storybook/StoryExample';
+import { VerticalGroup } from '../Layout/Layout';
+
+import { TextLink } from './TextLink';
+import mdx from './TextLink.mdx';
+
+const meta: Meta = {
+  title: 'General/TextLink',
+  component: TextLink,
+  parameters: {
+    docs: {
+      page: mdx,
+    },
+    controls: { exclude: ['href', 'external'] },
+  },
+  argTypes: {
+    variant: { control: 'select', options: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'body', 'bodySmall', undefined] },
+    weight: {
+      control: 'select',
+      options: ['bold', 'medium', 'light', 'regular', undefined],
+    },
+    color: {
+      control: 'select',
+      options: ['primary', 'secondary', 'disabled', 'link', 'maxContrast', undefined],
+    },
+    inline: { control: 'boolean' },
+  },
+};
+
+export const Example: StoryFn = () => {
+  return (
+    <VerticalGroup>
+      <StoryExample name="This is a 'inline + external' link with the default behaviour">
+        <TextLink href="https://google.es" inline={true} external>
+          Go to Google
+        </TextLink>
+      </StoryExample>
+      <StoryExample name="This is a 'standalone + external' link with the default behaviour">
+        <TextLink href="https://grafana.com/docs/grafana/latest/" inline={false} external>
```

Comment:
To me there is no real difference shown (besides of the colour) between the two TextLink examples in this story. I assumed that when using the inline prop the use case would like this: 
Lorem ipsum dolor sit amet, [consectetuer](http://example.com/) adipiscing elit.
So it would make sense to have something like this as an example.

Reply:
Good idea @L-M-K-B! Thanks!